jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/property/TestSQLCast.java

```diff
@@ -0,0 +1,47 @@
+package org.eclipse.persistence.jpa.test.property;
```

Comment:
License header...

Reply:
Woops, missed that! Thanks for the review!